solutions/cses-1640.mdx

```diff
@@ -118,6 +118,45 @@ int main() {
 ```
 
 </CPPSection>
+	
+<JavaSection>
+	
+```java
+import java.io.*;
+import java.util.*;
+
+public class SumOfTwoValues {
+	public static void main (String[] args) throws IOException {
```

Comment:
You don't need a throws statement because you're using standard input and output
```suggestion
	public static void main (String[] args) {
```